examples/dummy-iiostream.c

```diff
@@ -368,6 +368,10 @@ int main (int argc, char **argv)
 				size_t sample_size = fmt->length / 8 * repeat;
 
 				buf = malloc(sample_size * buffer_length);
+				if (!buf) {
+					perror("out of memory, malloc failed\n");
```

Comment:
perror() will append the string corresponding to the errno which more or less says the same thing. Maybe mention the function where this fails or "CHANNEL_READ".